src/resources/elements/ConnectButton/ConnectButton.ts

```diff
@@ -6,11 +6,15 @@ import { autoinject, containerless, customElement, singleton } from "aurelia-fra
 
 import { DisposableCollection } from "services/DisposableCollection";
 import { EventAggregator } from "aurelia-event-aggregator";
-import { EventConfigTransaction } from "services/GeneralEvents";
+import { EventConfigException, EventConfigTransaction } from "services/GeneralEvents";
 import { TransactionReceipt } from "services/TransactionsService";
 import { Utils } from "services/utils";
 import { bindable } from "aurelia-typed-observable-plugin";
 
+const SAFE_APP_ERROR_CODE = 200;
```

Comment:
dont suppose we handle all such consts in one place, should this be in need of changing ?

Reply:
ideally yes, but since we don't right now, imo, the next best place is on top of the file.
I'll add a todo comment